src/data/ProxySource.ts

```diff
@@ -0,0 +1,17 @@
+import semver from 'semver';
+
+import DocsSource from './DocsSource';
+
+const branchBlacklist = new Set(['docs', 'v8', 'v11', 'v12', 'v13']);
+export default new DocsSource({
+	id: 'proxy',
+	name: 'proxy',
+	global: 'proxy',
+	docsRepo: 'discordjs/docs',
+	repo: 'discordjs/discord.js',
+	defaultTag: 'stable',
+	branchFilter: (branch: string) => !branchBlacklist.has(branch) && !branch.startsWith('dependabot/'),
+	tagFilter: (tag: string) =>
+		tag.includes('@discordjs/proxy') &&
+		semver.gt(tag.replace(/(^@.*\/.*@v?)?(?<semver>\d+.\d+.\d+)-?.*/, '$<semver>'), '0.2.0'),
```

Comment:
This will need to be changed once the package is released, and likely with `semver.gte`/`semver.ge` to include the minimum version.